lib/linguist/languages.yml

```diff
@@ -3372,6 +3372,7 @@ Python:
   color: "#3572A5"
   extensions:
   - ".py"
+  - ".py3"
```

Comment:
Done. Maybe worth mentioning it in the md. I was confused by the `py` at the top.